src/Module/BaseApi.php

```diff
@@ -195,17 +204,30 @@ private static function getUserByBearer(string $authorization)
 		return $token['uid'];
 	}
 
+	/**
+	 * Get the application record via the proved request header fields
+	 *
+	 * @return array application record
+	 */
 	public static function getApplication()
 	{
-		$redirect_uri = !isset($_REQUEST['redirect_uri']) ? '' : $_REQUEST['redirect_uri'];
-		$client_id    = !isset($_REQUEST['client_id']) ? '' : $_REQUEST['client_id'];
+		$redirect_uri  = !isset($_REQUEST['redirect_uri']) ? '' : $_REQUEST['redirect_uri'];
+		$client_id     = !isset($_REQUEST['client_id']) ? '' : $_REQUEST['client_id'];
+		$client_secret = !isset($_REQUEST['client_secret']) ? '' : $_REQUEST['client_secret'];
```

Comment:
Simplification;
```suggestion
		$redirect_uri  = $_REQUEST['redirect_uri']  ?? '';
		$client_id     = $_REQUEST['client_id']     ?? '';
		$client_secret = $_REQUEST['client_secret'] ?? '';
```

Reply:
There are several places in the code in this structure. Question is if I should change them in a single batch?

Reply:
In the same file, yes. Across multiple files, in a separate PR, please.